data/variable/database/templates/users.mysql.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE users (
 
     id       INTEGER UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
     username VARCHAR(50),
-    digesta1 VARCHAR(32),
+    digesta1 VARCHAR(60),
```

Comment:
We are not using MD5. We are using `password_*` API. I used `digesta1` because we rely on the sabre/dav schemas.